testnets/_IBC/initia-prysmdevnet.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "initia",
```

Comment:
there is no initiatestnet registered. Does the prysmtestnet connect to initia mainnet?